docs/admin/admission-controllers.md

```diff
@@ -418,15 +418,15 @@ subresource of the referenced *owner* can change it.
 {% assign for_k8s_version="v1.9" %}{% include feature-state-alpha.md %}
 The `PVCProtection` plugin adds the `kubernetes.io/pvc-protection` finalizer to newly created Persistent Volume Claims (PVCs). In case a user deletes a PVC the PVC is not removed until the finalizer is removed from the PVC by PVC Protection Controller. Refer to the [PVC Protection](/docs/concepts/storage/persistent-volumes/#persistent-volume-claim-protection) for more detailed information.
 
-### PersistentVolumeLabel
+### PersistentVolumeLabel (DEPRECATED)
 
 This admission controller automatically attaches region or zone labels to PersistentVolumes
 as defined by the cloud provider (for example, GCE or AWS).
 It helps ensure the Pods and the PersistentVolumes mounted are in the same
 region and/or zone.
 If the admission controller doesn't support automatic labelling your PersistentVolumes, you
 may need to add the labels manually to prevent pods from mounting volumes from
-a different zone.
+a different zone. PersistentVolumeLabel is DEPRECATED in favor of PersistentVolumeLabelController in [cloud controller manager](/docs/tasks/administer-cluster/running-cloud-controller/).
```

Comment:
Thanks for the revision. In the revised version, `PersistentVolumeLableController` is still confusing to users (readers). The page pointed did mentioned new ways of labelling persistent volumes, however, it is about creating and using Initializers. Could you please revise this again to make sure users (not necessarily Go developers) to understand what needs to be done?

Reply:
What about current version?

Description about PersistentVolumeLableController is removed. But `cloud conroller manager` is keeped.